test/forwarderFactory.js

```diff
@@ -9,7 +9,7 @@ const BigNumber = require('bignumber.js');
 const Forwarder = artifacts.require('./Forwarder.sol');
 const ForwarderFactory = artifacts.require('./ForwarderFactory.sol');
 
-const ForwarderABI = require('../ABIs/Forwarder.json');
```

Comment:
should this be on a separate ABIs folder?

Reply:
that is forwarder.json

Reply:
I removed the ABI folder in another PR since it was only being used inside the test folder and didn't contain any other ABIs. I can move this file to a resources folder inside of /tests

Reply:
Also doesn't hardhat automatically provide the ABI as part of the artifact? Shouldn't we use that instead of statically generate ABIs and use them in test?

Reply:
they do, just didnt feel like refactoring old tests

Reply:
let's do the refactor then lol!